src/csharp.tmLanguage.yml

```diff
@@ -2164,7 +2164,7 @@ repository:
           (?:\s*\[(?:\s*,\s*)*\]\s*)* # array suffix?
         )
       )?
-      \b(\g<identifier>)\b\s*
+      \s+(\g<identifier>)\b\s*
```

Comment:
Ditto here. I'm not sure why this is necessary here. I would expect this to be after the type.

Reply:
After the type spaces get consumed by the type pattern, but the optional pattern (one line above) doesn't ensure a space needed to create a correct word boundary for `@`.